apps/src/constants.js

```diff
@@ -40,6 +40,9 @@ exports.TestResults = {
   EMPTY_FUNCTIONAL_BLOCK: 18,    // There's a functional block with an open input
   EXAMPLE_FAILED: 19,            // One of our examples didn't match the definition
 
+  // start using negative values, since we consider >= 20 to be "solved"
+  EMPTY_FUNCTION_NAME: -1,       // We have a variable or function with the name ""
```

Comment:
:rocket: Guess we didn't anticipate how many ways things can go wrong!

Out of curiosity, what goes wrong if we change the value of `TOO_MANY_BLOCKS_FAIL`, etc?  Do we have to touch a lot of places?